hil-test/Cargo.toml

```diff
@@ -254,7 +284,7 @@ generic-queue = [
 integrated-timers = [
     "esp-hal-embassy/integrated-timers",
 ]
-octal-psram = ["esp-hal/octal-psram", "dep:esp-alloc"]
+octal-psram = ["esp-hal/octal-psram", "esp-alloc"]
```

Comment:
`dep:` meant there was no implicit `esp-alloc` feature, which I need in the new test.